core/validation/validation.js

```diff
@@ -128,10 +128,14 @@ export const getFieldValidation = field => R.pathOr(newInstance(), [keys.fields,
 
 export const getErrors = R.propOr([], keys.errors)
 export const hasErrors = R.pipe(getErrors, R.isEmpty, R.not)
+export const isError = validation =>
+  hasErrors(validation) || R.pipe(getFieldValidations, R.values, R.any(isError))(validation)
```

Comment:
does this work on a nested validation object? (e.g. category) (same for isWarning)

Reply:
maybe it's better to have 2 functions, one like this, ```isError``` that checkes if there are any errors in the validation object, another one that calls isError recursively, I don't know how to call it, something like ```hasErrorsDeep```